### ISSUE TITLE
Trying to install on Windows 10

### ISSUE THREAD

**Issue by bshea5** [SanderMertens/bake#59]
I'm running `bake setup` via an admin level cmd, but keep getting the message below. Any ideas how to finish the setup?

![image](https://user-images.githubusercontent.com/3984775/66175010-bb6d2880-e60c-11e9-890f-d3af27be4411.png)

**Comment by SanderMertens** [comment]
Hmm, that is strange. Can you try rerunning bake setup with `--debug`?

If that doesn't work, you can run the setup like this:

```
bake setup --local
```

That will not install anything to your global environment. Instead, the bake environment will be located in ~/bake. With local installs, you'll have to manually set PATH to the bake environment.

**Comment by bshea5** [comment]
Adding --debug only revealed the following of note:

![image](https://user-images.githubusercontent.com/3984775/66249315-30606100-e6e6-11e9-9101-9ee09c3a435e.png)

Running it locally makes it available once added to PATH of my cmd or termnial.

**Comment by SanderMertens** [comment]
Bake does need a valid visual studio environment. This sounds like a similar issue as the one reported here:  https://mail.google.com/mail/u/0/#inbox/FMfcgxwDrbnzBSgSKnqKNJMpFNFNgjGh (scroll down to the last comments)

TL;DR: bake installs a script which invokes the visual studio prompt for you, but for some reason that doesn't seem to play well with local installs.
